dp/cloud/go/services/dp/storage/diff.go

```diff
@@ -0,0 +1,54 @@
+package storage
+
+import (
+	"math"
+)
+
+const (
+	radiusM            = 6371e3
+	distanceThresholdM = 10
+)
+
+func ShouldENodeBDUpdate(prev *DBCbsd, next *DBCbsd) bool {
+	return canUpdate(prev, next) &&
+		(paramsChanges(next, prev) || coordinatesChanged(next, prev))
+}
+
+func canUpdate(prev *DBCbsd, next *DBCbsd) bool {
+	return !prev.CpiDigitalSignature.Valid && next.CbsdCategory.String == "A"
```

Comment:
&& indoor_deployment == true. Also "a" should be lower case

Reply:
As discussed removed category completely